### ISSUE TITLE
Add 'include' functionality

### ISSUE THREAD

**Issue by macxcool** [RasmusLindroth/i3keys#35]
It would be very cool if this could handle 'include' statements in the config to pull in other config files with key definitions.

**Comment by elder-n00b** [comment]
Include statements should work, do you have a case where they don't?
Note that environment variables like `$HOME` will have to be written `${HOME}` in the config file or they won't expand.

**Comment by macxcool** [comment]
My include lines look like this:
`include includes/keys_workspaces`
I'm just specifying the subfolder, not the full path. Is that why there's a problem?

**Comment by elder-n00b** [comment]
It's a possibility. See if specifying the correct absolute path makes a difference (after reloading sway config **and** restarting i3keys).
note: i3keys reads the config at startup (like Sway), it needs restarting if the config is changed.

I just tried adding an `include xyz`, where `xyz` has a `bindsym` directive, and i3keys correctly picked up the new binding.

Just to make sure I understand correctly, your `includes/keys_workspaces` works in Sway but the bindings therein are not detected by i3keys?

**Comment by elder-n00b** [comment]
Interesting, I'll try look into it some more. If I manage to reproduce the problem, probably I can fix it too and PR.
P.S. thank Rasmus, he wrote it. I'm just passing by.

**Comment by RasmusLindroth** [comment]
I think I've found the problem.
It seems the program only gets the path for sway and not i3
https://github.com/RasmusLindroth/i3keys/blob/99e368e4cbb4f82f4e9461c3fd43054add0c3c37/i3parse/parse.go#L241

So the fix should be quite easy, just create a function equal to `helpers.GetSwayDefaultConfig() ` and send the name of the window manager down to the parse function. I'll look into it later and test it, but in the meantime absolute paths should work :)

**Comment by macxcool** [comment]
Cool. I'll await your fix and test it on my setup. I'm in no big hurry. ;-)

**Comment by RasmusLindroth** [comment]
@macxcool Do you have sway installed on your computer? Because now that I look at the code it should be picking up i3 folders if you don't have it installed

**Comment by RasmusLindroth** [comment]
Could you test this branch and see it it works?

https://github.com/RasmusLindroth/i3keys/tree/i3-include-test

```bash
cd /tmp
git clone https://github.com/RasmusLindroth/i3keys.git
cd i3keys
git checkout i3-include-test
go build
/tmp/i3keys web 8080
```

**Comment by macxcool** [comment]
Yes. I _do_ have sway installed, just for experimenting with. I run i3, normally, though.

I will test that this evening (i.e. in a couple of hours)

**Comment by macxcool** [comment]
That works. I see all my configured bindings and my modes and everything! Cool.
